examples/tv-casting-app/linux/main.cpp

```diff
@@ -114,6 +115,30 @@ HelpOptions helpOptions("tv-casting-app", "Usage: tv-casting-app [options]", "1.
 
 OptionSet * allOptions[] = { &cmdLineOptions, &helpOptions, nullptr };
 
+static void OnBindingAdded(const EmberBindingTableEntry & binding)
+{
+    if (binding.type == EMBER_MULTICAST_BINDING)
```

Comment:
instead of == should we check for `!= EMBER_UNICAST_BINDING` instead?

If you do that, then:
  - we handle other types (e.g. EMBER_MANY_TO_ONE_BINDING ... or even UNUSED_BINDING if ever there)
  - can decrease the indent below
  
 Or do you plan to implement group binding support?
Why is this not a switch/ifelse to at least log_progress if an unknown/unsupported is received?

Reply:
cleaned up